ui.apps/src/main/content/jcr_root/apps/acs-commons/config/com.adobe.acs.commons.users.impl.EnsureServiceUser-email-service.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" 
+jcr:primaryType="sling:OsgiConfig" 
```

Comment:
Please indent this file similar to the other OSGi configs

Reply:
Done. 👍